meson.build

```diff
@@ -1,6 +1,7 @@
 project('mt32emu', 'cpp',
         version : '2.4.2',
         license : 'LGPL-2.1-or-later',
+        meson_version : '>=0.53.0',
```

Comment:
@eli-schwartz Thank you! I had no idea about setting up minimum meson version in project function (and Meson generating helpful warnings in result) - it's extremely useful! I updated dosbox-staging meson.build to employ this technique already.

As for going to `>= 0.53.0` - initially I intended to replace dictionary with a table of tables, which would allow for `>= 0.49.0`… but after a bit of consideration - I am ok with keeping it as it is, just to keep the wrap cleaner.

If minimum version 0.53.0 works for @FeignInt then it works for me as well.

Debian Stable is at 0.49.0, but Testing has 0.56.0 already and we can expect new Stable release in ~4-5 months. Fedora and Arch have up-to-date Mesons already. I am not ok with going past 0.53.0 due to Ubuntu 20.04 LTS.

Reply:
@FeignInt implemented a clever solution that makes it unnecessary to have dict or table-of-tables here:
https://github.com/mesonbuild/mt32emu/blob/af9cba11a8f0057c1400c2f0e74a7f264af6bb6b/include/mt32emu/c_interface/meson.build#L2

Does `string : <value>` has some special meaning here? Won't this break when building on Windows?

Reply:
I have since revised this to
```
'@0@'.format(src_file).split('/')
```
getting us [ 'path', 'to', 'file.ext' ]
instead of [ 'string : path', 'to', 'file.ext' ] :)

it was never a 'real' problem, since only using last bit *output : file_name[-1]*